src/game/FACTIONS.ts

```diff
@@ -0,0 +1,8 @@
+const FACTIONS: {
+  [key: string]: string;
```

Comment:
As this is a list that will never change we can type it with the values and users of the type will get error suggestions on things like lowercase vs uppercase, or wrong options.
`[key: string]: 'ALLIANCE' | 'HORDE';`